app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\User;
+
+class UserController extends Controller
+{
+    public function __construct(){
+        $this->middleware('auth');
+    }
+
+    public function members(){
+        $members=User::where('id', '!=', auth()->user()->id)->get();
```

Comment:
Please make it a habit to add a space before and after the `=`
```suggestion
        $members = User::where('id', '!=', auth()->user()->id)->get();
```

So it's easier to read.